src/System.CommandLine/Symbol.cs

```diff
@@ -95,7 +95,7 @@ private protected void AddArgumentInner(Argument argument)
 
         public SymbolSet Children { get; } = new SymbolSet();
 
-        public void AddAlias(string alias)
+        public void AddAlias(string? alias)
```

Comment:
>? [](start = 35, length = 1)

Since this method will throw on a `null` input, I'd mark as not nullable. This will help callers not pass a `null` value in.